_includes/tutorials/ksql-heterogeneous-json/ksql/markup/dev/problem-description.adoc

```diff
@@ -0,0 +1,34 @@
+Let's say you have a topic `source_data`, that contains JSON.  But each nested JSON object has a different structure.  Addtionally, in each object there is a mix of types within the values.
+
+They each have one field in common that you want to pull out in a query and you don't care about the structure of the individual JSON objects
```

Comment:
Given the example below, can we name the field that is common?

Reply:
changed the text some LMK WYT